datafusion/physical-expr/src/equivalence/properties.rs

```diff
@@ -2212,6 +2212,97 @@ mod tests {
             );
         }
 
+        Ok(())
+    }
+    #[test]
+    fn test_eliminate_redundant_monotonic_sorts() -> Result<()> {
+        let schema = Arc::new(Schema::new(vec![
+            Field::new("a", DataType::Date32, true),
```

Comment:
Would it be possible to use the same names in the schema `(a, b, c)` that are used in the test description `(date, ticker, time)`? I found it hard to mentally map between the two while reviewing this code

Reply:
Ah yeah sorry that's a typo by me